rma/wizard/stock_picking_return_views.xml

```diff
@@ -14,6 +14,10 @@
                         name="create_rma"
                         attrs="{'invisible': [('picking_type_code', '!=', 'outgoing')]}"
                     />
+                    <field
+                        name="rma_operation_id"
+                        attrs="{'invisible': [('create_rma', '=', False)], 'required': [('create_rma', '=', True)]}"
```

Comment:
Not sure we should require it here.

Reply:
The RMA created from the return wizard is confirmed right away. The user will get an error if they don't fill in the operation. So, I think it's more user-friendly to make the field required in the view